pipelines/nextflow/subworkflows/genome_prepare/genome_prepare.nf

```diff
@@ -61,39 +64,47 @@ workflow GENOME_PREPARE {
- 
+
             PROCESS_GFF3(UNPACK_FILE.out.uncompressed_file, CHECK_JSON_SCHEMA_GENOME.out.verified_json, UNPACK_FILE.out.gca)
-            ch_gene_models = PROCESS_GFF3.out.gca.concat(PROCESS_GFF3.out.gene_models)
- 
+
             CHECK_JSON_SCHEMA_FUNCT(PROCESS_GFF3.out.functional_annotation, PROCESS_GFF3.out.gca)
-            ch_functional_anno = CHECK_JSON_SCHEMA_FUNCT.out.gca.concat(CHECK_JSON_SCHEMA_FUNCT.out.verified_json)
- 
+
             GFF3_VALIDATION(PROCESS_GFF3.out.gene_models)
- 
+
             PROCESS_FASTA_PEP(DOWNLOAD_ASM_DATA.out.protein_fa, DOWNLOAD_ASM_DATA.out.genome_gbff, DOWNLOAD_ASM_DATA.out.gca, '1')
-            ch_process_pep = PROCESS_FASTA_PEP.out.gca.concat(PROCESS_FASTA_PEP.out.processed_fasta)
         }
- 
+
         PROCESS_SEQ_REGION(CHECK_JSON_SCHEMA_GENOME.out.verified_json, DOWNLOAD_ASM_DATA.out.asm_report, DOWNLOAD_ASM_DATA.out.genome_gbff, DOWNLOAD_ASM_DATA.out.gca)
         CHECK_JSON_SCHEMA_SEQREG(PROCESS_SEQ_REGION.out.seq_region, PROCESS_SEQ_REGION.out.gca)
-        ch_seq_region = CHECK_JSON_SCHEMA_SEQREG.out.gca.concat(CHECK_JSON_SCHEMA_SEQREG.out.verified_json)
- 
+
         PROCESS_FASTA_DNA(DOWNLOAD_ASM_DATA.out.genome_fna, DOWNLOAD_ASM_DATA.out.genome_gbff, DOWNLOAD_ASM_DATA.out.gca, '0')
-        ch_process_dna = PROCESS_FASTA_DNA.out.gca.concat(PROCESS_FASTA_DNA.out.processed_fasta)
- 
-        AMEND_GENOME_DATA(CHECK_JSON_SCHEMA_GENOME.out.verified_json, DOWNLOAD_ASM_DATA.out.asm_report, DOWNLOAD_ASM_DATA.out.genome_gbff, DOWNLOAD_ASM_DATA.out.gca, params.brc4_mode)
- 
-        standardised_files = Channel.of(ch_genome, ch_gene_models, ch_functional_anno, ch_process_pep, ch_seq_region, ch_process_dna)
-        standardised_files.map{it -> [accession, it]}.groupTuple().view()
-}
 
+        AMEND_GENOME_DATA(CHECK_JSON_SCHEMA_GENOME.out.verified_json, DOWNLOAD_ASM_DATA.out.asm_report, DOWNLOAD_ASM_DATA.out.genome_gbff, DOWNLOAD_ASM_DATA.out.gca, params.brc4_mode)
+        amended_json = AMEND_GENOME_DATA.out.amended_json
+        // amended_json.view()
+
+        // Group files
+        // prepared_files = CHECK_JSON_SCHEMA_GENOME.out.verified_json
+        //             .concat(PROCESS_GFF3.out.gene_models, CHECK_JSON_SCHEMA_FUNCT.out.verified_json,
+        //             PROCESS_FASTA_PEP.out.processed_fasta, CHECK_JSON_SCHEMA_SEQREG.out.verified_json,
+        //             PROCESS_FASTA_DNA.out.processed_fasta)
+        //             .map{it -> [accession, it]}
+        //             .groupTuple()
+
+        // // Collect in manifest, checks and stats
+        // collect_dir = COLLECT_FILES(prepared_files)
+        // manifest_dired = MANIFEST(collect_dir, accession)
+        // manifest_checked = CHECK_INTEGRITY(manifest_dired, params.brc4_mode)
+        // manifest_stated = MANIFEST_STATS(manifest_checked, 'datasets')
+
+        // // Publish the data
+        // PUBLISH_DIR(manifest_stated, output_dir, accession)
```

Comment:
I assume the pipeline is not finished at this point?

Reply:
I think that's not going to work unless you keep the accession associated with each file along the whole pipeline (that's the only way to make sure we keep them in sync). This is really tricky and I'll need to focus on this once I'm done with other things...

That said, you can push the PR and we can work on that next part afterwards.